Assets/LeapMotionModules/GraphicRenderer/Scripts/LeapGraphic.cs

```diff
@@ -8,7 +8,7 @@ namespace Leap.Unity.GraphicalRenderer {
 
   [ExecuteInEditMode]
   [DisallowMultipleComponent]
-  public abstract partial class LeapGraphic : MonoBehaviour, ISpaceComponent {
+  public abstract partial class LeapGraphic : MonoBehaviour, ISpaceComponent, IEquatable<LeapGraphic> {
```

Comment:
I could have sworn that I did, but I can't replicate it anymore :( I only added it because I saw something in the profiler, and making this change made it go away, but I have no idea what it is anymore...